apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -581,6 +581,7 @@ export const TicketPageContent = ({ ticket, pollCommentsQuery, refetchTicket, or
             ticketId: id,
         },
         skip: !persistor,
+        fetchPolicy: 'cache-and-network',
```

Comment:
why?

Reply:
@nomerdvadcatpyat You've already implemented comment polling properly, why do you need to use network again here?